src/Sylius/Bundle/CoreBundle/Form/Type/ProductReviewAdminType.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\CoreBundle\Form\Type;
+
+use Sylius\Component\Core\Model\Customer;
+use Sylius\Component\Core\Model\Product;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+/**
+ * @author Daniel Richter <nexyz9@gmail.com>
+ * @author Mateusz Zalewski <mateusz.zalewski@lakion.com>
+ * @author Grzegorz Sadowski <grzegorz.sadowski@lakion.com>
+ */
+class ProductReviewAdminType extends ProductReviewType
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        parent::buildForm($builder, $options);
+
+        $builder
+            ->add('reviewSubject', 'entity', [
+                'class' => Product::class,
+                'label' => 'sylius.form.review.product',
+                'property' => 'name',
+            ])
+            ->add('author', 'entity', [
+                'class' => Customer::class,
+                'label' => 'sylius.form.review.author',
+                'property' => 'email',
+            ])
+        ;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function configureOptions(OptionsResolver $resolver)
+    {
+        $resolver->setDefaults([
+            'rating_steps' => 5,
+            'data_class' => $this->dataClass,
+            'validation_groups' => $this->validationGroups,
+        ]);
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getName()
+    {
+        return sprintf('sylius_%s_review_admin', $this->subject);
```

Comment:
But this is ``ProductReviewAdminType``, not ``ProductReviewType``, form for administration page is quite different.